locales/en/messages.json

```diff
@@ -3651,6 +3651,9 @@
     "firmwareFlasherNoFirmwareSelected": {
         "message": "<b>No firmware selected to load</b>"
     },
+    "firmwareFlasherNoTargetsLoaded": {
+        "message": "<b>No targets loaded</b>"
```

Comment:
Maybe the `<b>` can be applied outside the translation? This is a short string, but translators sometimes forgot the tags or don't close them, etc. So if it can go out of the messages file, better.